rdmd_test.d

```diff
@@ -347,15 +354,20 @@ void runTests()
     assert(res.status == 0, res.output);
     assert(!res.output.canFind("compile_force_src"));
 
-    auto fullCompilerPath = environment["PATH"]
-        .splitter(pathSeparator)
-        .map!(dir => dir.buildPath(compiler ~ binExt))
-        .filter!exists
-        .front;
+    // for absolute compiler path, we make the path relative again
+    string fullCompilerPath = void;
+    if (compiler.isAbsolute)
```

Comment:
I guess technically this wasn't necessary since `buildPath` with an absolute path will always result in that absolute path, but I guess this is more efficient and results in a better error if the path doesn't exist.